subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CheckstylePlugin.java

```diff
@@ -45,7 +45,7 @@
  */
 public class CheckstylePlugin extends AbstractCodeQualityPlugin<Checkstyle> {
 
-    public static final String DEFAULT_CHECKSTYLE_VERSION = "8.37";
+    public static final String DEFAULT_CHECKSTYLE_VERSION = "8.45.1";
```

Comment:
Please add a note in the upgrade guide - see the commit that bumped to 8.37 for inspiration.

Reply:
Fixed